users/urls.py

```diff
@@ -1,9 +1,12 @@
 from django.urls import path
-from django.contrib.auth import views as auth_views
 from . import views
 
 urlpatterns = [
-    path('register', views.register, name='register'),
-    path('login', auth_views.LoginView.as_view(template_name='users/login.html'), name='login'),
-    path('logout', auth_views.LogoutView.as_view(), name='logout'),
+    path("register", views.RegisterView.as_view(), name="register"),
+    path("changepassword", views.ChangePasswordView.as_view(),
+         name="change-password"),
+    path("login", views.LoginView.as_view(), name="login"),
+    path("logout", views.LogoutView.as_view(), name="logout"),
+    path("me/profile", views.ProfileView.as_view(), name="profile"),
+    path("me/balance", views.BalanceView.as_view(), name="balance"),
```

Comment:
should be another file

Reply:
didn't get the use of having seperate files for urls?

Reply:
done